pom.xml

```diff
@@ -1853,6 +1853,17 @@
             <url>https://maven.restlet.com</url>
         </repository>
         -->
+        <repository>
+            <id>4science-nexus</id>
+            <url>https://nexus.4science.it/repository/maven-public</url>
+            <layout>default</layout>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
```

Comment:
This will be removed once the https://github.com/DSpace/orcid-jaxb-api/pull/3 is merged and the new artifact is pushed to maven central

Reply:
@abollini : As of today, https://github.com/DSpace/orcid-jaxb-api/pull/3 has been merged, and [v3.0.0](https://github.com/DSpace/orcid-jaxb-api/releases/tag/orcid-jaxb-api-3.0.0) was tagged & released to Maven Central.  It should appear in Maven Central in the next hour or so.